config/routes.rb

```diff
@@ -216,6 +212,9 @@
   end
   get '/jobs', to: redirect('/')
 
+  # Catchall route matching any routes coming from React, EncodeDashboard.js
+  match '*path', to: 'encode_records#index', via: :all
```

Comment:
Can we make this matcher more specific?

Reply:
I think React routes should work with normal route configurations. I put this just to catch all the routes that aren't matched in the routes above.